obc/blinkProgram.md

```diff
@@ -0,0 +1,77 @@
+
+![Cap1](https://user-images.githubusercontent.com/60119461/94093690-d58ab280-fdeb-11ea-8142-550277b76909.PNG)
+
+Open the program and a screen like the one below should appear. From here select “Access to MCU selector” and a list of MCU’s will open.
+
+![Cap2](https://user-images.githubusercontent.com/60119461/94093693-d885a300-fdeb-11ea-997b-b484aee455ec.PNG)
+
+From this list look under the reference column for “STM32F030R8Tx” and “LQFP64” under the package column. Once the right MCU is found, click “start project” at the top right. If everything was done right, the screen below should appear.
+
+![Cap3](https://user-images.githubusercontent.com/60119461/94093696-d9b6d000-fdeb-11ea-9b9c-895761d325e8.PNG)
+
+![Cap5](https://user-images.githubusercontent.com/60119461/94093703-dde2ed80-fdeb-11ea-89be-4c2ff8559e80.PNG)
+
+Before anything else is done click on the “System Core” tab, and then the “SYS” option. Check off the “Debug Serial Wire” box and the click the drop-down menu for the “Timebase Source” and click “TIM1.”. After this the pins are ready to be configured. 
+Each pin has its own unique, location and name dedicated to itself. These names can be matched up with the physical MCU itself. The name of each pin should be available if the mouse is hovered over the pins. Each of these pins can all have specific function. For example, VDD provides the voltage to the chip and VSS provides the ground. To modify the function of a pin click it and a drop-down menu will appear. For the purposes of the blink program, click “GPIO_Output” option on pin 29. Note: This can be configured to be any pin but for it to match up with further steps picking pin 29 will make things easier. After this, click on the Project Manager tab found at the top.
+
+
+![Cap6](https://user-images.githubusercontent.com/60119461/94093711-e20f0b00-fdeb-11ea-84d1-d94d2084c7cd.PNG)
+
+“Project Name” will allows a title to be given to the project and Project Location” controls where on the computer the generated code to be saved. The “Toolchain / IDE” drop down menu will allow you to specify which IDE the code will be generated. In this case the “TrueSTUDIO” option will be selected. Now click “GENERATE CODE” at the top right and say accept any prompts for further downloads. The program should generate a folder containing code based on the specifications made. Open the file made and open “. project.” Once in Atollic find the project in the project explorer and click the drop down, and then click the drop down on “Src” where “main.c” can be seen. Double click on the “main.c” and the code should be visible. 
+
+## Programing
+With the code now visible, it is time to edit it.  For this specific program only 2 functions will be added, HAL_GPIO_TogglePin(), and HAL_Delay(). 
+

```

Comment:
I am assuming that these screenshots are from the associated STM32 Hal documentation. If you have the link to the pdf it might be worth throwing it in.